### DISCUSSION TITLE
Cannot render PDF using Resource Route

### DISCUSSION THREAD

**Discussion by ukcw** [remix-run/remix#5439] [Q&A]
I'm trying to render a PDF stored in my `/public` folder but I get "Failed to load pdf document" error. I've looked at [this](https://github.com/remix-run/remix/discussions/4463)  prior discussion but the solution doesn't seem to work for me. I've attached images of my code for reference.

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/34738958/218398287-e28650da-ab70-49e0-bf18-370ec88269f8.png">
<img width="1716" alt="image" src="https://user-images.githubusercontent.com/34738958/218398349-e754ce17-0de0-458b-947f-cac4b395ce0c.png">
<img width="1727" alt="image" src="https://user-images.githubusercontent.com/34738958/218398399-5441a1bf-978c-40d6-ba4f-965784c297ed.png">

**Comment by machour** [comment]
Your path for `readFileSync` is not correct.

That API read from your file system, without knowing about your project, so `/foobar` is the "foobar" file located at the root of your file system.

Try a relative path instead, like "public/My_Resume.pdf", or "../public/My_Resume.pdf" (add ../ until it works)

**Comment by ukcw** [comment]
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/34738958/218533682-f2a7a2d2-69ae-4bb9-8185-207a4e28a956.png">

I've adjusted to use a relative path but it still doesn't work for me. I get this error: 
```
Unexpected Server Error
TypeError: (void 0) is not a function
```

The commented out code (the `iframe`) works though.
